### ISSUE TITLE
Deployment Issues

### ISSUE THREAD

**Issue by smcgregor** [responsible-ai-collaborative/aiid#847]
This is an issue associated with the current staging version immediately prior to deployment. I'll highlight issues according to whether they are blocking (i.e., "don't deploy until these are fixed"), non-blocking (i.e., "fix these after deployment"), and Blocking Feature Announcement (i.e., things that need to be done before we can make the translation feature show).

# Blocking Deployment

* Khoa ingested two Spanish reports (one on incident 188 and another on 186). For 188, "dry run" was not set to false, so it did the filler translation. I then tried to fix it in the UI. When you edit a Spanish-language report on the English citation page, you edit the source Spanish report and not the English translation. This means I cannot correct a bad translation if I want to.
* We have nothing on the card indicating it was translated from Spanish. We need a banner across the cards indicating it was machine translated, likely with a link to switch to the Spanish version of the site.

# Non-blocking

* The width of the page on initial render jumps between widths. I believe it is rendering a default, then updating to whatever the screen size is. This is particularly clear when it comes to the search box text, which changes between "Search over 1600 reports of AI harms" and "Search 1600+ reports".
* Language selection could have flags in addition to the text for selecting the language 🏴󠁧󠁢󠁥󠁮󠁧󠁿🇪🇸. This may be super annoying to implement given how we turn on/off languages, in which case please feel free to ignore this one.
* Is `Descubrir` and `Enviar` the right translations of the discover and submit applications? @cesarvarela knows better than I do, but I would have thought the imperative form would be good here (e.g., `descubre`)
* @lmcnulty I get "Could not compute semantic similarity" very often, which I think is a quality of service problem pre-dating this deployment. The API either needs to be made more reliable from the user perspective, or the user-messaging needs to be more friendly.

# Blocking Feature Announcement

* Markdown links often kick the user back to English. For example, https://staging-aiid.netlify.app/es/taxonomies
* This should be translated to Spanish: https://staging-aiid.netlify.app/es/blog/raic-holds-first-all-hands-meeting
* I need to write the blog post talking about the feature for the announcement
* Markdown issue on this page, and it ends rather suddenly: https://staging-aiid.netlify.app/es/about We don't currently have the resources required to translate all these pages manually beyond Spanish, and we certainly don't want to translate to all supported languages in the future every time the documentation is updated. I think we need to develop a pattern where untranslated pages are presented in the English, but there is a component that displays the option of viewing the page in a machine translated version (see, for example, the language toggle here: https://montrealethics-ai.translate.goog/?_x_tr_sl=en&_x_tr_tl=fr&_x_tr_hl=en-US ). Most people will not care to read the documentation pages in their first language, or at least they should typically have the capacity of reading it in English. In the future we can look to develop a volunteer base to manage these translations, but this should be a best-effort solution.
* It looks like Markdown issues are pretty common: https://staging-aiid.netlify.app/es/about_apps
* The "Último Reporte" component on the front page is not presenting the translated text and clicking into it places you in the English version again.
* This should not be translated: "McGregor, S. (2021) Prevención de fallas repetidas de IA en el mundo real mediante la catalogación de incidentes: la base de datos de incidentes de IA. En Actas de la Trigésima Tercera Conferencia Anual sobre Aplicaciones Innovadoras de la Inteligencia Artificial (IAAI-21). Conferencia Virtual." https://staging-aiid.netlify.app/es/research

**Comment by smcgregor** [comment]
@cesarvarela I added two blocking issues for the top of the heap.

**Comment by smcgregor** [comment]
Translation notification content:

* It should have a slightly different color in its background to indicate it is not part of the content itself, it should say something like, "This report has been translated from LANGUAGE to CURRENT_LANGUAGE. Learn more"

**Comment by cesarvarela** [comment]
- [ ] Khoa ingested two Spanish reports (one on incident 188 and an....
- [ ] The width of the page on initial render jumps between widths...
- [ ] Language selection could have flags in addition to the text for selecting the language 🏴󠁧󠁢󠁥󠁮󠁧󠁿🇪🇸. ...
- [ ] Is Descubrir and Enviar the right translations of the discover and submit applications
- [ ] I get "Could not compute semantic similarity" very often, which I think is a quality of service...
- [ ] Markdown links often kick the user back to English. For example, https://staging-aiid.netlify.app ...
- [ ] Markdown issue on this page, and it ends rather suddenly: https://staging-aiid.netlify.app/es/about ...
- [ ] We don't currently have the resources required to translate all these pages manually beyond Spanish, and we certainly don't want to translate to all supported languages in the future every time the documentation is updated. I think we need to develop a pattern where untranslated pages are presented in the English, but there is a component that displays the option of viewing the page in a machine translated version (see, for example, the language toggle here: https://montrealethics-ai.translate.goog/?_x_tr_sl=en&_x_tr_tl=fr&_x_tr_hl=en-US ). 
- [ ] It looks like Markdown issues are pretty common: https://staging-aiid.netlify.app/es/about_apps
- [ ] The "Último Reporte" component on the front page is not presenting the translated text and clicking into it places you in the English version again.
- [ ] This should not be translated: "McGregor, S. (2021) Prevención de fallas repetidas de IA en el mundo real mediante la catalogación de incidentes: la base de datos de incidentes de IA. En Actas de la Trigésima Tercera Conferencia Anual sobre Aplicaciones Innovadoras de la Inteligencia Artificial (IAAI-21). Conferencia Virtual." https://staging-aiid.netlify.app/es/research

**Comment by cesarvarela** [comment]
Everything here has been fixed or it is mentioned in another issue
